src/mscorlib/shared/System/Reflection/ReflectionTypeLoadException.cs

```diff
@@ -39,6 +40,38 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
             info.AddValue("Exceptions", LoaderExceptions, typeof(Exception[]));
         }
 
+        public override string Message
+        {
+            get
+            {
+                if (LoaderExceptions.Length == 0)
+                {
+                    return base.Message;
+                }
+
+                StringBuilder text = new StringBuilder();
+                text.AppendLine(base.Message);
+                foreach (Exception e in LoaderExceptions)
+                {
+                    text.AppendLine(e.Message);
+                }
+                return text.ToString();
+            }
+        }
+
+        public override string ToString()
+        {
+            StringBuilder text = new StringBuilder();
+            text.AppendLine(SR.ReflectionTypeLoad_LoadFailed);
```

Comment:
@wtgodbe Was there a reason why you have hard coded this message error instead of using the message that the exception was constructed with?

Reply:
I may have misunderstood what AggregateException was doing here - https://github.com/dotnet/coreclr/blob/eb2e9f90ba3679fa8386c4130f81c8061e8d4ca4/src/mscorlib/shared/System/AggregateException.cs#L452. Should I just use `base.ToString()` instead?

Reply:
or `base.Message`?

Reply:
`base.ToString()` is probably the most appropriate one to use. I have made the change in the CoreRT mirror PR. You do not need to worry about it.